Icon/Icon.js

```diff
@@ -58,11 +58,23 @@ const IconBase = kind({
 		 * @default 'large'
 		 * @public
 		 */
-		size: PropTypes.oneOf(['small', 'large', 'huge'])
+		size: PropTypes.oneOf(['small', 'large', 'huge']),
+
+		/**
+		 * The amount of sprite "cells" in the src image.
+		 *
+		 * This property only affects image-based icons, not icon glyphs.
+		 *
+		 * @type {Number}
+		 * @default 0
```

Comment:
There's a disconnect between the documented and actual default below. Which is correct?

Reply:
`1` is correct. This should be fixed. `0` should never be used, but if you don't have an image, it doesn't really matter what this value is. I'll update the docs.